include/deal.II/lac/solver_qmrs.h

```diff
@@ -74,86 +94,101 @@ class SolverQMRS : public Solver<VectorType>
   /**
    * Standardized data struct to pipe additional data to the solver.
    *
-   * There are two possibilities to compute the residual: one is an estimate
-   * using the computed value @p tau. The other is exact computation using
-   * another matrix vector multiplication.
+   * The user is able to switch between right and left preconditioning, that means
+   * solving the systems <i>P<sup>-1</sup>A</i> and <i>AP<sup>-1</sup></i> respectively,
+   * using the corresponding parameter. Note, that left preconditioning means to
+   * employ the preconditioned (BiCG-)residual and otherwise the unpreconditioned one.
+   * The default is the application from the right side.
    *
-   * QMRS, is susceptible to breakdowns, so we need a parameter telling us,
+   * The solver_tolerance threshold is used to define the said bound below which the residual
+   * is computed exactly. See class documentation for more information. The default value is 1e-9,
+   * that is the default solving precision multiplied by ten.
+   *
+   * SQMR is susceptible to breakdowns (divisions by zero), so we need a parameter telling us,
    * which numbers are considered zero. The proper breakdown criterion is very
-   * unclear, so experiments may be necessary here.
+   * unclear, so experiments may be necessary here. It is even possible to achieve convergence
+   * despite of dividing through small numbers. There are even cases, in which it is advantageous to
+   * accept such divisions because the cheap iteration cost makes the algorithm the fastest of all
+   * available indefinit solvers. Nonetheless, the default breakdown threshold value is 1e-16.
    */
   struct AdditionalData
   {
     /**
      * Constructor.
      *
-     * The default is no exact residual computation and breakdown parameter
+     * The default is right preconditioning and breakdown parameter
```

Comment:
The default is the use right preconditioning, with the @p solver_tolerance chosen to be 1e-9 and the @p breakdown_threshold set at 1e-16.